tiny_dnn/util/macro.h

```diff
@@ -12,3 +12,9 @@
 #if defined _WIN32 && !defined(__MINGW32__)
 #define CNN_WINDOWS
 #endif
+
+#ifdef _MSC_VER
+#define CNN_BE_INLINE_ALREADY __forceinline
+#else
+#define CNN_BE_INLINE_ALREADY __attribute__((always_inline)) inline
+#endif
```

Comment:
Oka let me revert it back. 
![image](https://cloud.githubusercontent.com/assets/10494087/26750599/d36df030-4844-11e7-8b49-3c478b8f88ad.png)